components/GitHubEventAction.tsx

```diff
@@ -63,6 +63,16 @@ function GitHubEventAction({type, payload}: GitHubEventActionProps): JSX.Element
       }
       return <span>{action} <ELink link={prURL}>pull request #{prNum}</ELink> in</span>;
     }
+    case 'PullRequestReviewCommentEvent': {
+      const action = payload.action;
+      const prNum = payload.comment.number;
```

Comment:
I don't see a `payload.comment.number` in the GitHub API. You will probably need to enter the `pull_request` field and get the number from there.

Reply:
Yea, I think I was looking at the wrong documentation. I ended us just using the JSON outta the endpoint. Seems to have worked.